apps/github-cascading-app/src/cascading/interfaces.ts

```diff
@@ -0,0 +1,30 @@
+/** Describes the minimal interface that a logger need to implement to be used by the Cascading plugin */
+export type BaseLogger = { debug: (log: string) => void; info: (log: string) => void; warn: (log: string) => void; error: (log: string) => void };
```

Comment:
Why not using the @o3r/logger definitions ?

Reply:
Because it is not matching the interface, the logger provided by probot is only single string parameter